includes/Integrations/Core_Themes_Support.php

```diff
@@ -93,7 +143,19 @@ public function init() {
 			return;
 		}
 
+		$this->style_handle = 'web-stories-theme-style-' . get_stylesheet();
```

Comment:
Does this need to be a class property?

I'd just use `'web-stories-theme-style-' . get_stylesheet()` directly in `assets()`

Reply:
Have kept it as class property because we need it in two places, in `assets()` to register it and then we enqueue it while rendering the stories.

Reply:
Let's just get rid of `assets()` and call `$this->register_style( $this->style_handle, [ Renderer::STYLE_HANDLE ] );` in `embed_web_stories()`

Don't see the need for separate registration